dcs-packaging-tool/dcs-packaging-tool-gui/src/main/java/org/dataconservancy/packaging/gui/presenter/impl/BasePresenterImpl.java

```diff
@@ -46,8 +46,6 @@ protected void bindBaseElements() {
         view.getHeaderViewAboutLink().setOnAction(arg0 -> view.showAboutPopup());
         
         view.getHeaderViewHelpLink().setOnAction(arg0 -> view.showHelpPopup());
-        
-        view.getCancelLink().setOnAction(arg0 -> getController().goToPreviousPage());

```

Comment:
Indeed, the wording of a lot of the things I put in should be looked at and changed as needed, like helps and stuff. I basically put place holders in.